migration/migration_blackbox_test.go

```diff
@@ -454,6 +455,34 @@ func testMigration67(t *testing.T) {
 	assert.NotNil(t, parentID)
 }
 
+func testMigration69(t *testing.T) {
+	// migrate to previous version
+	migrateToVersion(sqlDB, migrations[:(initialMigratedVersion+24)], (initialMigratedVersion + 24))
+	// fill DB with data
+	assert.Nil(t, runSQLscript(sqlDB, "069-create-workitems.sql"))
+	// then apply the change
+	migrateToVersion(sqlDB, migrations[:(initialMigratedVersion+25)], (initialMigratedVersion + 25))
+	// check the value of execution_order
+	workitem1, err := uuid.FromString("00000000-0000-0000-0000-000000000004")
+	require.Nil(t, err)
+	workitem2, err := uuid.FromString("00000000-0000-0000-0000-000000000005")
+	require.Nil(t, err)
+	workitem3, err := uuid.FromString("00000000-0000-0000-0000-000000000006")
+	require.Nil(t, err)
+	expectations := make(map[uuid.UUID]float64)
+	expectations[workitem1] = 1000
+	expectations[workitem2] = 2000
+	expectations[workitem3] = 1000
+	for workitemID, expectedExecutionOrder := range expectations {
+		var executionOrder float64
+		stmt, err := sqlDB.Prepare("select execution_order from work_items where id=$1")
+		require.Nil(t, err)
+		_ = stmt.QueryRow(workitemID.String()).Scan(&executionOrder)
+		require.NotNil(t, executionOrder)
+		assert.Equal(t, expectedExecutionOrder, executionOrder)
```

Comment:
@hectorj2f I create two workitems in space1 and one workitem in space2. The expected execution order after migration script runs should have been:
workitem1 - 1000 (space1)
workitem2 - 2000 (space1)
workitem3 - 1000 (space2)

But the actual execution order that i got was:
workitem1 - 11000 (space1)
workitem2 - 12000 (space1)
workitem3 - 11000 (space2)

The db cleans up after running all the tests in the migration black box tests. There were a few workitems which were from previous tests. So, in my migration test script, i create new workitems only after clearing the previous data -> https://github.com/fabric8-services/fabric8-wit/pull/1478/files/ddf7309c4ff100b4338041ef49af72e943044b2b#diff-932bde01d0de92b23051f157cc9e93e3R4